From d0a4e145bd5af3b35cfaa8df1ef94cdcd1b0ff59 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Thu, 29 Sep 2011 09:22:07 +0800
Subject: [PATCH 0440/2463] ENGR00158456-1 ipuv3 dev: return error for split
 mode with rotation

Currently we do not support split mode with rotation.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 include/linux/ipu.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/ipu.h b/include/linux/ipu.h
index d6c1588..17638ab 100644
--- a/include/linux/ipu.h
+++ b/include/linux/ipu.h
@@ -237,6 +237,7 @@ enum {
 	IPU_CHECK_ERR_SPLIT_INPUTH_OVER,
 	IPU_CHECK_ERR_SPLIT_OUTPUTW_OVER,
 	IPU_CHECK_ERR_SPLIT_OUTPUTH_OVER,
+	IPU_CHECK_ERR_SPLIT_WITH_ROT,
 };
 
 /* IOCTL commands */
-- 
1.7.7.4

